pulp_container/app/registry_api.py

```diff
@@ -895,6 +892,20 @@ def put(self, request, path, pk=None):
 
         content_data = json.loads(raw_data)
 
+        try:
```

Comment:
There are also other places where you can use manifest schema validation and get rid of the extra if/else. I'll take the conversation offline.

Reply:
same for Sync codepath